mne/io/brainvision/brainvision.py

```diff
@@ -64,12 +69,14 @@ class RawBrainVision(_BaseRaw):
     @verbose
     def __init__(self, vhdr_fname, montage=None,
                  eog=('HEOGL', 'HEOGR', 'VEOGb'), misc=(),
-                 scale=1., preload=False, response_trig_shift=0, verbose=None):
+                 scale=1., preload=False, response_trig_shift=0,
```

Comment:
Can we deprecate `response_trig_shift`? Can't this be done using `events[:, 0] += response_trig_shift` after calling `find_events`? Seems like this is a bit of constructor argument pollution in that sense. It also doesn't exist in other readers despite often wanting to shift triggers, so I'm not sure why `brainvision` should have it specifically. Is there something in the file format that's sufficiently different from others to justify it?